contrib/ansible/vagrant/provision.sh

```diff
@@ -0,0 +1,9 @@
+echo "hello, here is  a sample provisioning script that demonstrates everything works"
+ls /vagrant
+### Required for ansible??
+### Otherwise 'no yum module installed'...
+sudo yum install -y yum # needed for the ansible python stuff
```

Comment:
This should be somehow fixed in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/contrib/ansible/roles/pre-ansible/tasks/main.yml#L11

I strongly strongly object to node provision scripts. That's how the salt stuff turned to what it is today.